index.js

```diff
@@ -68,7 +69,33 @@ function toggleColor(element) {
   }
 }
 
+const header = document.querySelector("h1#header");
+header.addEventListener("click", function () { toggleColor(header) });
+
 /***** Deliverable 2 *****/
 
+const likeButton = document.querySelector(".like-button");
+const likeParagraph = document.querySelector(".likes");
+
+likeButton.addEventListener("click", function(){
+  const likes = parseInt(likeParagraph.textContent);
+  const addLikes = likes + 1;
+  likeParagraph.textContent = `${addLikes} likes`; 
+});
 
 /***** Deliverable 3 *****/
+const form = document.querySelector("#new-goal-form");
+form.addEventListener("submit", function(event) {
+  event.preventDefault();
+
+  const goalLink = event.target.link.value
+  const goalDescription = event.target.description.value
+
+  const newGoal = {
+    link: goalLink,
+    description: goalDescription,
+  }
+  renderGoal(newGoal);
```

Comment:
Nice work! If you want to save on variables declarations, you could also create your object like so:
```javascript
  const newGoal = {
    link: event.target.link.value,
    description: event.target.description.value
  }
```

A way of getting the id to update is to get it from the latest goal in the goals array in the player object in `data.js`.